### DISCUSSION TITLE
New triagers

### DISCUSSION THREAD

**Discussion by AA-Turner** [sphinx-doc/sphinx#11882] [Project Announcements]
I have invited the following to have triage rights in Sphinx -- there are no formal expectations for the said rights, but the list has been compiled on the basis of recent & sustained helpful contributions, and so may be useful for discussions on the issue tracker, etc.

- @jayaddison 
- @jdillard
- @mgeier
- @jbms

The above people may of course decline the invitation!

A

**Comment by jayaddison** [comment]
@AA-Turner @picnixz @danieleades @chrisjsewell I think I may take a break from triaging and contributing for a few weeks - it's enjoyable and there's more I'd like to learn/review/merge, but I've found it a bit difficult to keep pace.

I'd like to support stability for a v7.3.0 release, so I'll be back again after some time to recuperate.

**Reply by chrisjsewell** [reply]
Thanks for letting us know,
yeh obviously there is absolutely no expectations on your time so go enjoy life,
but it certainly helps to have this feedback on loosely when people may or may not be available
